Sources/Orbit/Components/Text.swift

```diff
@@ -64,7 +67,7 @@ public struct Text: View, FormattedTextBuildable {
     }
 
     @ViewBuilder private var selectableLabelWrapper: some View {
-        if isSelectable {
+        if textIsCopyable {
```

Comment:
If you want to stick with copyable, you should probably rename the rest of "selectable" here.

Reply:
Updated